src/core_plugins/timelion/public/webpackShims/jquery-flot.js

```diff
@@ -1,7 +1,16 @@
+const $ = require('jquery');
+
+if (window) window.jQuery = $;
```

Comment:
Another option that you might want to explore is the [imports-loader](https://github.com/webpack-contrib/imports-loader), which will prefix the code included in the bundle with something a require statement:

```js
require("imports-loader?jQuery=jquery!./jquery.flot");
```

produces something like this:

```js
/* 0 */
function () {
  __webpack_require__(2)
},
/* 1 */
function () {
  // jquery source
},
/* 2 */
function () {
  var jQuery = __webpack_require__(1)
  // original contents of ./jquery.flot
}
```